Community Plugin: Lineup Builder

James-Fallon opened the pull request:
<!--- Delete this section if submitting a theme -->
# I am submitting a new Community Plugin

## Repo URL

<!--- Paste a link to your repo here for easy access -->
Link to my plugin: https://github.com/James-Fallon/obsidian-lineup-builder

## Release Checklist

<!--- Confirm that you have done the following before submitting your plugin -->
- [x] I have tested this on Windows, macOS, and iOS
- [x] My GitHub release contains all required files
  - [x] `main.js`
  - [x] `manifest.json`
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] README clearly describes the plugins purpose and provides clear usage instructions.
- [x] I have added a license in the LICENSE file.

lishid commented:
https://github.com/James-Fallon/obsidian-lineup-builder/blob/33b52fd983c708e24d79329a359f7b2feb90b329/src/svg-builder.ts#L26
Here you run the risk of a DOM injection attack - you are directly putting in user-provided strings to the HTML text that is then used to render the DOM. You should either sanitize all user inputs before it makes it into the HTML string, or use DOM building functions to create elements and append text into it.

James-Fallon commented:
@lishid Thanks for the comment, good spot. I have implemented encoding of user input on your suggestion, as seen in this commit: https://github.com/James-Fallon/obsidian-lineup-builder/commit/548a9251c6ed0b3200ba273cfe54eb37b411ebff

I've since added a new release version 1.0.1 with the fix and updated the manifest.json and version file accordingly.

lishid commented:
Sorry it took so long for the review!
